src/sentry/replays/endpoints/organization_replay_count.py

```diff
@@ -50,7 +57,11 @@ def get(self, request: Request, organization: Organization) -> Response:
             )
         except NoProjects:
             return Response({})
-        data_source = Dataset.IssuePlatform if request.GET.get("data_source") else Dataset.Discover
+
+        result = ReplayDataSourceValidator(data=request.GET)
+        if not result.is_valid():
+            raise ParseError(result.errors)
+        data_source = Dataset.IssuePlatform if result.validated_data else Dataset.Discover
```

Comment:
need to check if the key of `source` on `result.validated_data` is equal to `Dataset.IssuePlatform` -- right now this will always return `Dataset.IssuePlatform` which is why the tests below are failing